test/e2e/network/netpol/network_policy.go

```diff
@@ -686,10 +683,6 @@ var _ = common.SIGDescribe("Netpol", func() {
 							// don't use named ports
 							Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 80},
 						},
-						{
```

Comment:
note to self in case we ever revitalize dns policy tests this needs to be added back :)